addon/components/frost-object-browser.js

```diff
@@ -50,6 +51,16 @@ export default Ember.Component.extend({
       return {}
     }
   },
+  
+  /**
+   * Verifies the state of showCountInSubTitle variable before showing the subtitle
+   * (in case someone doesn't need the counts)
+   */
+  @readOnly
+  @computed('subtitle', 'showCountInSubTitle', 'computedValuesTotal')
+  computedSubtitle (subtitle, showCountInSubTitle, count) {
```

Comment:
OK, but I don't understand why we have to do it like this while the code right below is written with the comment above the @readOnly